lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_download_translations_action.rb

```diff
@@ -0,0 +1,92 @@
+# This action is the new version of android_update_metadata (AndroidUpdateMetadataAction) and should now be used instead of that one
+
+module Fastlane
+  module Actions
+    class AndroidDownloadTranslationsAction < Action
+      def self.run(params)
+        require_relative '../../helper/android/android_localize_helper.rb'
+        require_relative '../../helper/git_helper.rb'
+
+        res_dir = File.join(ENV['PROJECT_ROOT_FOLDER'] || '.', params[:project_dir_name], 'src', 'main', 'res')
+
+        Fastlane::Helper::Android::LocalizeHelper.create_available_languages_file(
+          res_dir: res_dir,
+          locale_codes: [params[:source_locale]] + params[:locales].map { |h| h[:android] }
+        )
+        Fastlane::Helper::Android::LocalizeHelper.download_from_glotpress(
+          res_dir: res_dir,
+          glotpress_project_url: params[:glotpress_url],
+          locales_map: params[:locales]
+        )
+
+        # Update submodules then lint translations
+        Fastlane::Helper::GitHelper.update_submodules()
+        Action.sh('./gradlew', params[:lint_task])
+
+        Fastlane::Helper::GitHelper.commit(message: 'Update translations', files: res_dir, push: true)
```

Comment:
Out of scope for this PR, but this line reminded me of something I've been meaning to suggest for a while. I feel it would be nice to have some sort of dry-run functionality and/or to opt-out from pushing. What do you think? Should I write this down in a formalized RFC?

Reply:
I actually suggested the exact same idea when I talked about the work on toolkit with @jkmassel recently 👍 Great minds I guess … 😛

Will probably only keep that for a separate PR but definitively worth keeping in mind (maybe as a GH issue, or a Platform Request)?

Reply:
Request opened for this dry-run idea via paaHJt-1Zs-p2